Source/WTF/wtf/RunLoop.h

```diff
@@ -96,7 +96,7 @@ class WTF_CAPABILITY("is current") RunLoop final : public GuaranteedSerialFuncti
 #endif
     WTF_EXPORT_PRIVATE static Ref<RunLoop> create(ASCIILiteral threadName, ThreadType = ThreadType::Unknown, Thread::QOS = Thread::QOS::UserInitiated);
 
-    static bool isMain() { return main().isCurrent(); }
+    static bool isMain() { SUPPRESS_UNCOUNTED_LOCAL return main().isCurrent(); }
```

Comment:
I guess this is Sam's point that it would be clearer to annotate `isCurrent` as ignored at its declaration site. Maybe we should add that feature.

Reply:
Maybe! Oops, this should have been SUPPRESS_UNCOUNTED_ARG, not SUPPRESS_UNCOUNTED_LOCAL. Will fix.